src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -963,14 +963,6 @@ async function initializeRuntime(
   }
 
   initializeEnvironmentalVariables(frb);
-  if (process.env.GOOGLE_APPLICATION_CREDENTIALS) {
```

Comment:
TODO(me): We need to re-instate this warning but only if the user provided the credentials themselves.